op-node/node/config.go

```diff
@@ -154,6 +156,9 @@ func (cfg *Config) Check() error {
 	if err := cfg.Rollup.Check(); err != nil {
 		return fmt.Errorf("rollup config error: %w", err)
 	}
+	if cfg.Rollup.ProbablyMissingPectraBlobSchedule() {
+		return ErrMissingPectraBlobSchedule
+	}
```

Comment:
If this returns an error early, and if it then gets ignored by config flag, then it ends up skipping all the config-checks that come after this.

I think we should put the `IgnoreMissingPectraBlobSchedule` bool inside of the node config.

Reply:
Good catch. Yeah adding it as a bool field works, just tried to make the footprint of this as minimal as possible.